book/src/sprites/add_the_sprite_render_component.md

```diff
@@ -0,0 +1,82 @@
+# Add the SpriteRender Component
+
```

Comment:
Might wanna add some text here to explain and not just throw code at the user 😄

Reply:
user: "how do i include sprites?"
book: *\*!! code !!\**

haha, ya definitely gonna write something; had to disappear and wanted to push the changes